dht-node/.env.template

```diff
@@ -8,6 +8,10 @@ DB_PASSWORD=$DB_PASSWORD
 DB_DATABASE=gradido_community
 TYPEORM_LOGGING_RELATIVE_PATH=$TYPEORM_LOGGING_RELATIVE_PATH
 
+# Community
+COMMUNITY_NAME=$COMMUNITY_NAME
+COMMUNITY_DESCRIPTION=$COMMUNITY_DESCRIPTION
```

Comment:
how is that related to this PR?

Reply:
config values for the home-community entry in communities table